doc/api/util.md

```diff
@@ -552,8 +552,10 @@ If there is an `original[util.promisify.custom]` property present, `promisify`
 will return its value, see [Custom promisified functions][].
 
 `promisify()` assumes that `original` is a function taking a callback as its
-final argument in all cases, and the returned function will result in undefined
-behavior if it does not.
+final argument in all cases. If `original` is not a function, `promisify()` 
+will throw an error. If `original` is a function but its last argument is not a 
+Node.js style callback, it will be still be passed a Node.js style callback as its last
```

Comment:
If you or someone else would like a good second pull request after this one lands, I'd be all for getting rid of the awkward and non-obvious `Node.js style callback` terminology here and in `errors.md` and replacing it with `error-first callback` which is more informative, less awkward, and probably more in line with usage outside of our docs.

Reply:
Nit: long line, please add a line break at 80 chars. Also there seems to be a redundant "be"?